lib/ParameterGroups/ExecutiveParameterGroup.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace Heidelpay\PhpPaymentApi\ParameterGroups;
+
+/**
+ * This class provides every api parameter related to the company executive data
+ *
+ * @license Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ * @copyright Copyright © 2016-present heidelpay GmbH. All rights reserved.
+ *
+ * @link  http://dev.heidelpay.com/heidelpay-php-payment-api/
+ *
+ * @author  David Owusu
+ *
+ * @package heidelpay\php-payment-api\parameter-groups
+ */
+class ExecutiveParameterGroup extends AbstractParameterGroup
+{
+    /**
+     * Executive funktion
+     *
+     * @var string
+     */
+    public $function;
+
+    /**
+     * Executive salutation
+     *
+     * @var string
+     */
+    public $salutation;
+
+    /**
+     * Executive name given
```

Comment:
Maybe call it firstname for better understanding
```suggestion
     * Firstname of the executive 
```